qiskit/tools/visualization/_circuit_visualization.py

```diff
@@ -459,4 +467,5 @@ def _matplotlib_circuit_drawer(circuit,
     qcd = _matplotlib.MatplotlibDrawer(qregs, cregs, ops, scale=scale, style=style,
                                        plot_barriers=plot_barriers,
                                        reverse_bits=reverse_bits)
-    return qcd.draw(filename)
+    qcd.parse_circuit(circuit)
+    return qcd.draw(filename, verbose=True)
```

Comment:
Why is this being changed here? We should keep this PR to just modifying the latex path, it's complicated enough without bundling mpl changes in it.

Reply:
This shouldn't be here - the `parse_circuit()` function was merged into the `draw()` one in the mpl PR